internal/reader/home_chain.go

```diff
@@ -244,15 +243,10 @@ func (r *homeChainPoller) GetOCRConfigs(
 		"candidateConfig", allConfigs.CandidateConfig,
 	)
 
-	if allConfigs.ActiveConfig.ConfigDigest != [32]byte{} {
-		ocrConfigs = append(ocrConfigs, allConfigs.ActiveConfig)
-	}
-
-	if allConfigs.CandidateConfig.ConfigDigest != [32]byte{} {
-		ocrConfigs = append(ocrConfigs, allConfigs.CandidateConfig)
-	}
-
-	return ocrConfigs, nil
+	return ActiveAndCandidate{
+		ActiveConfig:    allConfigs.ActiveConfig,
+		CandidateConfig: allConfigs.CandidateConfig,
+	}, nil
```

Comment:
I think `allConfigs` is already of type `ActiveAndCandidate` right? Can just return directly

```suggestion
	return allConfigs, nil
```